.github/actions/binary-upload/action.yml

```diff
@@ -43,5 +43,5 @@ runs:
           | jq -r '.value' > "${AWS_WEB_IDENTITY_TOKEN_FILE}"
 
         for pkg in dist/*; do
-          aws s3 cp "$pkg" "${PYTORCH_S3_BUCKET_PATH}" --acl public-read
+          aws s3 cp "$pkg" "${PYTORCH_S3_BUCKET_PATH}" --acl public-read --checksum-algorithm SHA256
```

Comment:
I think the awscli version will need to be updated, I don't think --checksum-algorithm exists on 1.32.18

Reply:
Looks like the feature was released in 2.18.0.

https://github.com/aws/aws-cli/issues/6750#issuecomment-2474799767

Reply:
Alternatively, the lower level put object command can be used. But, the update might be better I reckon if it is feasible.

Reply:
I see a 1.36.40 on pip which does seem to have --checksum-algorithm

Reply:
I keep forgetting v2 is completely different.

Reply:
Updated awscli to `1.37.0`.